ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController.swift

```diff
@@ -376,10 +376,7 @@ public class BrowserViewController: UIViewController {
     // - Brave Rewards is enabled
     // - Brave News is enabled
     // - `ShouldAlwaysRunBraveAdsService` feature is enabled
```

Comment:
This comment is not needed because it was moved to shouldStartAds

Reply:
Fixed